azure-mgmt-servicefabric/azure/mgmt/servicefabric/__init__.py

```diff
@@ -0,0 +1,14 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Code generated by Microsoft (R) AutoRest Code Generator 1.2.2.0
```

Comment:
I agree it's weird, I filled an issue: https://github.com/Azure/autorest/issues/2499